scripts/kind-e2e/README.md

```diff
@@ -0,0 +1,111 @@
+# E2E environment
+
+The e2e environment can be used for local testing, development, and CI purposes.
+
+E2E environment consists of:
+- 3 k8s clusters deployed with [kind].
+  - Cluster1: One master node for broker and elasticsearch if required.
+  - Cluster{2..3}: One master node and two workers for gateways nodes.
+  The configuration can be viewed or changed at **scripts/kind-e2e/cluster{1..3}-config.yaml**.
+- Submariner installed and configured on top of the clusters.
+
+[kind] is a tool for running local Kubernetes clusters using Docker container “nodes”.
+
+## Prerequisites
+
+- [docker]
+- [kubectl]
+- [Add local user to docker group]
+
+Optional useful tools for troubleshooting:
+
+- [k9s]
+- [kubetail]
+
+## Installation and usage
+
+The environment can be used in two modes:
+
+### 1. Ephemeral/Onetime
+ 
+This mode will create e2e environment on a local workstation, run unit tests, E2E tests, and clean up the created resources afterwards.
+This mode is very convenient for CI purposes.
+    
+```bash
+make ci e2e
+```
+
+To test specific k8s version, additional **version** parameter can be passed to **make e2e** command.
+
+```bash
+make e2e version=1.13.7
+```
+
+Full list of supported k8s versions can found on [kind release page] page. We are using kind vesion 0.4.0.
```

Comment:
here and in other places: I don't like hard coding of information such as version numbers. These things will get out-dated in no time. No one will pay attention to update them.

Reply:
You are currently , currently we are working on a solution to that.